ui/components/app/edit-gas-fee-button/edit-gas-fee-button.js

```diff
@@ -28,13 +33,23 @@ export default function EditGasFeeButton({ userAcknowledgedGasMissing }) {
     return null;
   }
 
+  let icon = estimateUsed;
+  let title = estimateUsed;
+  if (
+    estimateUsed === PRIORITY_LEVELS.HIGH &&
+    editGasMode === EDIT_GAS_MODES.SWAPS
+  ) {
+    icon = 'swapSuggested';
+    title = 'swapSuggested';
+  }
+
```

Comment:
why not:
```suggestion
  if (
    estimateUsed === PRIORITY_LEVELS.HIGH &&
    editGasMode === EDIT_GAS_MODES.SWAPS
  ) {
    estimateUsed = 'swapSuggested';
  }

```
and then swap back `estimateUsed` for `icon` and `title` below

Reply:
`estimateUsed` is used at other placed also for example when updating transaction to get gas values, I would avoid changing it